Orange/widgets/unsupervised/owmds.py

```diff
@@ -187,135 +181,51 @@ def __init__(self):
         )
 
         form.addRow("Max iterations:",
-                    gui.spin(box, self, "max_iter", 10, 10 ** 4, step=1))
+                    gui.spin(box_optimization, self, "max_iter", 10, 10 ** 4, step=1))
```

Comment:
You can break this like this:

        form.addRow(
            "Max iterations:",
            gui.spin(box_optimization, self, "max_iter", 10, 10 ** 4, step=1))

to keep the lines after 80 characters. Do the same in the following lines for consistency.  This format makes the code more readable *in this particular case*, IMHO.